src/TextInput/TextInput.stories.tsx

```diff
@@ -140,6 +140,21 @@ Required.args = {
   required: true,
 }
 
+export const AssistiveText = Template.bind({})
+
+AssistiveText.args = {
+  id: 'textInput',
+  name: 'textInput',
+  placeholder: 'Enter name',
+  onChange: noop,
+  onInputChange: noop,
+  onBlur: noop,
+  label: 'Please enter your name',
+  outlined: true,
+  required: true,
```

Comment:
You forgot to add `renderAsTitle` for the story
```suggestion
  required: true,
  renderAsTitle: true,
```

Reply:
resolved in [82e535](https://github.com/marshmallow-insurance/smores-react/pull/2118/commits/82e535d107f5efa170c85b575cc570d45c8c77aa)